internal/docstore/docstore.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2019 The Go Cloud Authors
```

Comment:
Please update the copyright to "Go Cloud Development Kit Authors", here and everywhere.

Reply:
Done.